tests/functional/aws-node-sdk/test/legacy/tests.js

```diff
@@ -307,9 +307,9 @@ describe('aws-node-sdk test suite as registered user', function testSuite() {
         },
         { it: 'should get a range from the second part of an object ' +
             'put by multipart upload',
-            range: 'bytes=5242881-5242890',
+            range: 'bytes=5242880-5242889',
```

Comment:
Please add note that byte count starts at 0 so first part ends at 5242879 and second part starts at 5242880